packages/web-components/src/components/masthead/left-nav.ts

```diff
@@ -116,8 +116,12 @@ class DDSLeftNav extends StableSelectorMixin(BXSideNav) {
       const { expanded, _startSentinelNode: startSentinelNode, _endSentinelNode: endSentinelNode } = this;
       if (expanded) {
         this._hFocusWrap = focuswrap(this.shadowRoot!, [startSentinelNode, endSentinelNode]);
-      } else if (this._hFocusWrap) {
-        this._hFocusWrap = this._hFocusWrap.release();
+      } else {
+        this.querySelectorAll('[expanded]').forEach(ddsLeftNavMenu => ddsLeftNavMenu.removeAttribute('expanded'));
```

Comment:
Two suggestions, one is defining below static property and use it here:

```javascript
/**
 * A selector that will return expanded nav menus.
 */
static get selectorNavItems() {
  return `${ddsPrefix}-left-nav-menu[expanded]`;
}
```

Another is updating property instead of attribute (for the sake of type check), like `(ddsLeftNavMenu as DDSLeftNavMenu).expanded = false`.